src/lib/y2storage/proposal/autoinst_devices_planner.rb

```diff
@@ -52,7 +52,12 @@ def planned_devices(drives_map)
 
         drives_map.each_pair do |disk_name, drive_spec|
           disk = Disk.find_by_name(devicegraph, disk_name)
```

Comment:
We should use either `BlkDevice.find_by_name` or `devicegraph.disk_devices` here, since DADs are not disks.